source/minted.dtx

```diff
@@ -2405,77 +2405,116 @@
 % \end{macro}
 %
 %
+% \begin{macro}{minted@stylecounter}
+% We need a counter in order to map Pygments style names (which, if they contain digits or characters such as |-| or |_| can not serve directly in names of \LaTeX\ macros) to command prefixes.
+%    \begin{macrocode}
+\newcounter{minted@stylecounter}
+%    \end{macrocode}
+% \end{macro}
+%
+%
+% \begin{macro}{\minted@styleprefix}
+% This macro will act as a dictionary mapping Pygments style names to legal command prefixes for LaTeX macros.  Specifically they will be along the sequence |PYG| (default style), |PYGi|, |PYGii|, etc...
+%    \begin{macrocode}
+\newcommand{\minted@styleprefix}[1]{\csname minted@styleprefix@#1\endcsname}%
+%    \end{macrocode}
+% \end{macro}
+%
+%
 % \begin{macro}{\minted@checkstyle}
 % Make sure that style macros exist.
 %
-% We have to do some tricks with |\endlinechar| to prevent |\input| from inserting unwanted whitespace.  That is primarily for inline commands, where it would introduce a line break.  There is also the very unorthodox |\let\def\gdef| to make sure that macros are defined globally.  The catcodes for |-| and |_| must be changed during macro definition to accomodate style names like |paraiso-light|, |paraiso-dark|, and |algol_nu|.  The catcode for |%| must be changed to the default value in case \pkg{minted} is being used in a |.dtx| file; otherwise, \LaTeX\ comments in the Pygments style files will appear as literal text.
+% We have to do some tricks with |\endlinechar| to prevent |\input| from inserting unwanted whitespace.  That is primarily for inline commands, where it would introduce a line break.  There is also the very unorthodox |\let\def\gdef| to make sure that macros are defined globally.  The catcode for |%| must be changed to the default value in case \pkg{minted} is being used in a |.dtx| file; otherwise, \LaTeX\ comments in the Pygments style files will appear as literal text.
 %
 % If a style is not given, then revert to the |default| style, but create macros with prefix |PYG|, and create |default-pyg-prefix.pygstyle| if caching is on.  This allows a graceful fallback in the event that style is empty.  It is also purposefully used to create a complete set of macros with prefix |PYG|, so that the symbol macros may be used, as described next.
```

Comment:
This sentence needs update as now the name of the default style file is `default.pygstyle` rather.